pkg/segment/writer/metrics/metricssegment.go

```diff
@@ -246,7 +248,7 @@ func GetUnrotatedMetricStats(orgid uint64) (uint64, uint64, uint64) {
 
 	for _, m := range orgMetrics {
 		totalIncoming += m.bytesReceived
-		totalOnDisk += m.totalEncodedSize
+		totalOnDisk += m.mSegEncodeSize
```

Comment:
since we are calculating the total on variable X, just rename the total to be called totalX
in this case `totalMSegEncodeSize`